.github/workflows/test.yml

```diff
@@ -8,18 +8,25 @@ on:
 jobs:
   test:
     runs-on: ${{ matrix.platform }}
-    name: ${{ matrix.platform }} test
+    name: ${{ matrix.platform }} test node@${{ matrix.node }}
     strategy:
+      fail-fast: false
       matrix:
         platform: [ubuntu-latest, windows-latest]
-      fail-fast: false
+        node: [10, 12, 14, 16]
```

Comment:
We are now testing the created binary in all of the major node versions.

Reply:
Awesome